api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,1581 @@
+{
```

Comment:
<p>better to have a label on y-axis than a legend if a single data series is being plotted. also let's call it "yield (tonnes/hectare)" or whatever unit is being used. </p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done</p>